regimes/br/br.go

```diff
@@ -23,6 +23,9 @@ func New() *tax.RegimeDef {
 			i18n.EN: "Brazil",
 			i18n.PT: "Brasil",
 		},
+		Description: i18n.String{
+			i18n.EN: "TaxID represents either a CNPJ (for businesses) or CPF (for individuals), both of which are valid for issuing NFS-e invoices in Brazil.",
```

Comment:
So this is a description field for the regime and will be used in the documentation site here: https://docs.gobl.org/regimes/br

A quick fix would be to change the wording here so that it better fits the context.